docs/interfaces/modem/quectel.md

```diff
@@ -0,0 +1,170 @@
+        +-----------------------------------------------------------+
+        |--> Sending: AT+CFUN=1                                     |
+        +-----------------------------------------------------------+
+        |AT+CFUN=1                                                  |
+        +-----------------------------------------------------------+
+        |OK                                                   |
+        +-----------------------------------------------------------+
+        |--> Sending: AT+CGDCONT=1,"IP","internet.telenor.se"       |
+        +-----------------------------------------------------------+
+        |AT+CGDCONT=1,"IP","internet.telenor.se"                    |
+        +-----------------------------------------------------------+
+        |OK                                                         |
+        +-----------------------------------------------------------+
+        |--> Modem initialized.                                     |
+        +-----------------------------------------------------------+
+        |--> Sending: ATDT*99#                                      |
+        +-----------------------------------------------------------+
+        |--> Waiting for carrier.                                   |
+        +-----------------------------------------------------------+
+        |ATDT*99#                                                   |
+        +-----------------------------------------------------------+
+        |CONNECT 100000000                                          |
+        +-----------------------------------------------------------+
+        |--> Carrier detected.  Starting PPP immediately.           |
+        +-----------------------------------------------------------+
+        |--> Starting pppd at Mon Dec 16 12:43:43 2019              |
+        +-----------------------------------------------------------+
+        |--> Pid of pppd: 1317                                      |
+        +-----------------------------------------------------------+
+        |--> Using interface ppp0                                   |
+        +-----------------------------------------------------------+
+        |--> local  IP address 46.194.155.114                       |
+        +-----------------------------------------------------------+
+        |--> remote IP address 10.64.64.64                          |
+        +-----------------------------------------------------------+
+        |--> primary   DNS address 195.54.122.211                   |
+        +-----------------------------------------------------------+
+        |--> secondary DNS address 195.54.122.221                   |
+        +-----------------------------------------------------------+
+   - ping 8.8.8.8
+      - 64 bytes from 8.8.8.8: seq=0 ttl=55 time=55.250 ms
+ 
+ Or using pppd
+  - see https://github.com/hostmobility/mx4-wiki#modem
+      
+### Use GPS (simple commands)
+-------------------------
```

Comment:
Add code block syntax

Reply:
Well, it looks ok but you get a line instead of a block created using tripple backticks.